packages/form/addon/components/cf-field/input/radio.hbs

```diff
@@ -22,4 +22,11 @@
       {{option.label}}
     {{/if}}
   </label>
-{{/each}}
\ No newline at end of file
+{{/each}}
+{{#if (and (eq @field.question.raw.isRequired "false") @field.answer.value)}}
+  <br />
+  <a data-test-radio-reset href="#" {{on "click" (fn @onSave null)}}>{{t
+      "caluma.form.power-select.reset"
+    }}</a>
```

Comment:
Weird formatting

Reply:
Can't change what prettier is doin' here :man_shrugging:

Reply:
Yeah you can:
```hbs
<a data-test-radio-reset href="#" {{on "click" (fn @onSave null)}}>
  {{t "caluma.form.power-select.reset"}}
</a>
```
Prettier is not always prettier.